### PR TITLE
Update API version.

### PR DESCRIPTION
**Pull request by kegor**
Replicated and improved solution from https://github.com/groupdocs-total/GroupDocs.Total-for-NET-MVC/pull/81.

### CODE REVIEW

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 2
           

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pullRequest?prid=5133239&bid=16868104)

File: src/Products/Common/Resources/Resources.cs [hunk 1]
```diff
@@ -72,12 +71,12 @@ public ExceptionEntity GenerateException(System.Exception ex, String password)
...
             ExceptionEntity exceptionEntity = new ExceptionEntity();
             // Check if exception message contains password and password is empty
-            if (ex.Message.Contains("password") && String.IsNullOrEmpty(password))
+            if (ex.Message.ToLower().Contains("password") && String.IsNullOrEmpty(password))
             {
                 exceptionEntity.message = "Password Required";
             }
             // Check if exception contains password and password is set
-            else if (ex.Message.Contains("password") && !String.IsNullOrEmpty(password))
+            else if (ex.Message.ToLower().Contains("password") && !String.IsNullOrEmpty(password))
```
Comment (bobkovalex) [inline src/Products/Common/Resources/Resources.cs]: Codacy Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pullRequest?prid=5133239)

File: src/Products/Common/Resources/Resources.cs [hunk 2]
```diff
@@ -72,12 +71,12 @@ public ExceptionEntity GenerateException(System.Exception ex, String password)
             // Initiate exception
             ExceptionEntity exceptionEntity = new ExceptionEntity();
             // Check if exception message contains password and password is empty
-            if (ex.Message.Contains("password") && String.IsNullOrEmpty(password))
+            if (ex.Message.ToLower().Contains("password") && String.IsNullOrEmpty(password))
```
Comment (bobkovalex) [inline src/Products/Common/Resources/Resources.cs]: Codacy Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pullRequest?prid=5133239)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -5
         

Clones added
============
- src/Products/Editor/Controllers/EditorApiController.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pullRequest?prid=5133239&bid=16868104)

**Review by Pavel-Teplitsky — APPROVED** [review]

**Pull request merged**